src/Containers/Microsoft.NET.Build.Containers/Tasks/ComputeDotnetBaseImageTag.cs

```diff
@@ -25,33 +25,40 @@ public sealed class ComputeDotnetBaseImageTag : Microsoft.Build.Utilities.Task
     [Required]
     public string TargetFrameworkVersion { get; set; }
 
+    public string ContainerFamily { get; set; }
+
     [Output]
     public string? ComputedBaseImageTag { get; private set; }
 
     public ComputeDotnetBaseImageTag()
     {
         SdkVersion = "";
         TargetFrameworkVersion = "";
+        ContainerFamily = "";
     }
 
     public override bool Execute()
     {
         if (SemanticVersion.TryParse(TargetFrameworkVersion, out var tfm) && tfm.Major < FirstVersionWithNewTaggingScheme)
         {
             ComputedBaseImageTag = $"{tfm.Major}.{tfm.Minor}";
-            return true;
         }
-
-        if (SemanticVersion.TryParse(SdkVersion, out var version))
+        else if (SemanticVersion.TryParse(SdkVersion, out var version))
         {
             ComputedBaseImageTag = ComputeVersionInternal(version, tfm);
-            return true;
         }
         else
         {
             Log.LogError(Resources.Strings.InvalidSdkVersion, SdkVersion);
-            return false;
+            return !Log.HasLoggedErrors;
+        }
+        if (!string.IsNullOrEmpty(ContainerFamily))
```

Comment:
very minor: ensure you selected the proper test...

https://stackoverflow.com/questions/6976597/string-isnulloremptystring-vs-string-isnullorwhitespacestring

Reply:
Good catch, fixed just now.